src/hooks/use-deck-state.js

```diff
@@ -0,0 +1,88 @@
+import { useImmerReducer } from 'use-immer';
```

Comment:
Note: I've been meaning to rewrite this to get rid of the `use-immer` dependency- happy to PR that sooner rather than later

Reply:
If you want to PR that on monocle I can move that over. Immer is not a huge concern to me right now. So no rush.